Makefile

```diff
@@ -155,6 +155,23 @@ ELECTRON_BUILDER_OPTIONS += $(DISABLE_UPDATES_ELECTRON_BUILDER_OPTIONS)
 endif
 
 # ---------------------------------------------------------------------
+# Publish
+# ---------------------------------------------------------------------
+
+ifdef PUBLISH
+ifndef GH_TOKEN
+$(error No GitHub token found (GH_TOKEN is not set))
+endif
+ELECTRON_BUILDER_OPTIONS += --publish always
+ifeq ($(RELEASE_TYPE),production)
+# We want a human to push the final red button
+ELECTRON_BUILDER_OPTIONS += --draft
+else
+ELECTRON_BUILDER_OPTIONS += --prerelease
```

Comment:
I see, done! Notice that we're already guarding against invalid versions in the line 40 case.